CRM/PivotReport/DataActivity.php

```diff
@@ -7,41 +7,17 @@
  */
```

Comment:
With all the code you removed from here you don't need the AbstractGroup alias anymore. Please remove it